packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -989,7 +1000,7 @@ const runGenerator = async (
     })
 
     // Copy the base template either from the package or npm.
-    sh.cp('-rf', p.join(packagePath, '{*,.*}'), outputDir)
+    copyAllFiles(packagePath, outputDir)
```

Comment:
A similar problem here on Windows machine. The error looks like this:

![Arc 2025-04-30 at 22 17 40](https://github.com/user-attachments/assets/dbd97a45-1b40-4a1d-8e1c-5d178ba3ab28)